_posts/2017-10-02-learning-from-8thlight.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Learning from 8th Light, Day 1"
+date: 2017-10-02 10:03:00 +0200
+categories: how-we-learn 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 10
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-02-8thlight-1/8thlight.jpg
+---
+
+It all started at [BusConf], this August. Tobias Pflug (a colleague from HolidayCheck) asked [Daniel Irvine] (from [8th Light London]) if we could come over to London to learn from 8th Light about their apprenticeship program.
+Why? Because we want to be attractive to talent to stay competitive. And therefore we know we need to improve our skills continuously and keep producing high quality.
+Very simple :).
+
+[Daniel Irvine]: https://twitter.com/d_ir
+[BusConf]: http://www.bus-conf.org
+[8th Light London]: https://8thlight.com/locations/london/
+
+## Boarding the 8th Light plane
+
+So Tobias and me we boarded to London last Tuesday, to spend at least three full days with the 8th Lighters.
```

Comment:
- me -> I
- You can drop the "we" I think.
- That "last Tuesday" is quite some time ago already ;-)

Reply:
look at the date of the post :) ... i know you don't like those posts with timely relations like that, i think it gives them such a nice touch